roles/tpa_single_node/templates/manifests/spog/api/Deployment.yaml.j2

```diff
@@ -49,7 +49,7 @@ spec:
             - --crda-url
             - https://rhda.rhcloud.com/api/v4/analysis
             - --guac
-            - http://guac-graphql-pod:{{ tpa_single_node_guac_graphql_port }}/query
+            - https://{{ tpa_single_node_rhel_host }}:{{ tpa_single_node_guac_graphql_port }}/query
```

Comment:
@mrizzi  @gildub is it ok to expose ?

Reply:
@mrizzi the graphql is exposed outside the podman network, is reachable using the host ip